spec/factories/move.rb

```diff
@@ -120,6 +120,21 @@
       end
     end
 
+    trait :with_person_escort_record do
+      transient do
+        person_escort_record_status { 'unstarted' }
```

Comment:
adding a transient value will allow optionally passing in the `person_escort_record_status` when creating the factory